src/app/modules/admin/component/documents/documents.component.ts

```diff
@@ -31,80 +50,59 @@ export class DocumentsComponent implements OnInit {
 
   ngOnInit() {
     this.tableDocument = $('#document-table').DataTable(this.tableConfig);
-    this.documentService.getEntities().subscribe(decuments => {
-      this.addTableData(decuments);
-    });
+    $('#document-table tbody').on('click', '#first', this.selectFirstItem(this));
```

Comment:
'first' should not be id, because element is not unique. Make it as a class - `.first` etc.